@@@ A - classifier: DummyClassifier(constant=1, random_state=None, strategy='constant')
@@@ B - threshold stop at: 1
@@@ C - Repeat number: 1
/Users/author1/anaconda3/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:463: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
/Users/author1/anaconda3/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:464: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
/Users/author1/anaconda3/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:465: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
/Users/author1/anaconda3/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:466: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
/Users/author1/anaconda3/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:467: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
@@@ D - common_header:  {'F3-Artifact', 'F71-ogusakov@apache.org', 'F123', 'F110', 'F126', 'F71-hboutemy@apache.org', 'F71-ifedorenko@apache.org', 'F3-InputStream', 'F104', 'F25', 'F71-jdcasey@apache.org', 'F101', 'F3-int', 'F3-request.getLocalRepository()', 'F71-olamy@apache.org', 'F3-NA', 'F71-carlos@apache.org', 'F71-brianf@apache.org', 'F72', 'F120', 'F71-mkleint@apache.org', 'F115', 'F71-sisbell@apache.org', 'F15-public', 'F68', 'F77', 'F41', 'F71-jvanzyl@apache.org', 'F105', 'F3-ArrayList<Artifact>', 'F71-brett@apache.org', 'F15-private', 'F15-NA', 'F71-kenney@apache.org', 'F3-String', 'F22', 'F3-File', 'F117', 'F3-XmlPullParser', 'F71-bentmann@apache.org', 'F71-krosenvold@apache.org', 'F65', 'category', 'F3-ProjectSegment', 'F116', 'F3-null'}
Empty DataFrame
Columns: [F72, F25, F65, F68, F101, F104, F105, F3-NA, F3-ArrayList<Artifact>, F3-File, F3-InputStream, F3-request.getLocalRepository(), F3-null, F3-ProjectSegment, F3-XmlPullParser, F3-String, F3-int, F3-Artifact, F15-NA, F15-private, F15-public, F22, F116, F117, F120, F110, F123, F115, F77, F41, F126, F71-ogusakov@apache.org, F71-brett@apache.org, F71-hboutemy@apache.org, F71-kenney@apache.org, F71-sisbell@apache.org, F71-mkleint@apache.org, F71-carlos@apache.org, F71-olamy@apache.org, F71-jvanzyl@apache.org, F71-brianf@apache.org, F71-ifedorenko@apache.org, F71-bentmann@apache.org, F71-krosenvold@apache.org, F71-jdcasey@apache.org]
Index: []
Traceback (most recent call last):
  File "baseline.py", line 217, in <module>
    seed=42 + i, clf=clf, interesting_path=argv )
  File "baseline.py", line 43, in main
    training_x, training_y, testset_x, testset_y, training_canonical_ids, testing_canonical_ids = cleaning.data_clean(path, [interesting_path], seed)
  File "/Users/author1/repos/SA_public/src/cleaning.py", line 356, in data_clean
    training_x = pd.DataFrame(scaler.fit_transform(training_x), columns = training_x.columns)
  File "/Users/author1/anaconda3/lib/python3.6/site-packages/sklearn/base.py", line 571, in fit_transform
    return self.fit(X, **fit_params).transform(X)
  File "/Users/author1/anaconda3/lib/python3.6/site-packages/sklearn/preprocessing/_data.py", line 339, in fit
    return self.partial_fit(X, y)
  File "/Users/author1/anaconda3/lib/python3.6/site-packages/sklearn/preprocessing/_data.py", line 373, in partial_fit
    force_all_finite="allow-nan")
  File "/Users/author1/anaconda3/lib/python3.6/site-packages/sklearn/utils/validation.py", line 586, in check_array
    context))
ValueError: Found array with 0 sample(s) (shape=(0, 45)) while a minimum of 1 is required by MinMaxScaler.
